### ISSUE TITLE
Editor preferences suggestion

### ISSUE THREAD

**Issue by uliska** [wbsoft/frescobaldi#249]
I want to make the duration of the highlighting of matching brackets configurable.
David Nalesnik asked for longer or persistent highlighting here:
https://github.com/openlilylib/snippets/commit/c79128ecb98f107300591374e9858f6b3a8eed76#commitcomment-4417896
and I fully second this. In particular you're lost when the matching brackets aren't visible at the same time (which happens regularly, especially in Scheme functions).

While it's trivial to set this to a longer or persistent value it actually should have a preference. But I'm not sure where to put it. In the current Preferences dialog it would fit best to the Fonts and Colors page. 
But I suggest either creating a new Page with Editor Options or renaming the Fonts and Colors to Editor options because I have some more suggestions for editor preferences:
- Add an option to highlight text with background color that exceeds a specified line length (default 70 or 80). I have this in Eric and it's quite convenient when you're working on a wide screen that doesn't engourage you to keep your lines short.
- Make the duration of the point-and-click highlighting configurable too.

I'm sure there will be more ideas in the future.

---

Bottom line of this "Issue": Where should I place a new preference UI element?
